cafebabel/static/css/styles.css

```diff
@@ -2067,3 +2067,28 @@ footer p {
   top: 50%;
   left: 50%;
 }
+
+/* Error pages 404 & 500 */
```

Comment:
Also pinging @davidbgk on this: Shouldn't the page 500 also have it's own css within the page for the sake of not raising some unexpected error when importing an external file?